### PR TITLE
Z-Offset Parameter

### PR DESCRIPTION
**Pull request by Jmeyer1292**
Added a parameter that can be used to move robot blend paths into and out of the surface (along pose Z).

### CODE REVIEW

File: godel_surface_detection/src/services/blending_service_path_generation.cpp [hunk 1]
```diff
@@ -375,7 +378,9 @@ godel_surface_detection::TrajectoryLibrary SurfaceBlendingService::generateMotio
     nh.getParam(WINDOW_WIDTH_PARAM, scan_params.window_width);
     nh.getParam(MIN_QA_VALUE_PARAM, scan_params.min_qa_value);
     nh.getParam(MAX_QA_VALUE_PARAM, scan_params.min_qa_value);
-
+//    nh.getParam(Z_ADJUST_PARAM, scan_params.z_adjust);
```
Comment (jrgnicho) [inline godel_surface_detection/src/services/blending_service_path_generation.cpp:381]: Should this commented line be removed?

**Review by jrgnicho — CHANGES_REQUESTED** [review]

File: godel_surface_detection/src/services/blending_service_path_generation.cpp (hunk 1 above)
Comment (Jmeyer1292) [reply godel_surface_detection/src/services/blending_service_path_generation.cpp:381]: No

**Comment by jrgnicho** [comment]
The **surface_blending_service** crashes when I run the following demo:
```
roslaunch godel_irb2400_support irb2400_blending.launch real_pcd:=true pcd_location:=/absolute/path/to/pcd/file.pcd 
```

**Comment by Jmeyer1292** [comment]
I added fields to the saved blending parameter files. You need to go to your `~/.ros` folder and delete your saved godel parameters.

**Comment by jrgnicho** [comment]
@Jmeyer1292 deleting the godel parameters as you suggested worked.  This PR can be merged

**Review by jrgnicho — APPROVED** [review]

**Pull request merged**